srcpkgs/kernel-libc-headers/patches/no-rsync.patch

```diff
@@ -0,0 +1,14 @@
+diff --git Makefile Makefile
+index e30cf02..d023d8a 100644
+--- Makefile
++++ Makefile
+@@ -1277,8 +1277,7 @@ export INSTALL_HDR_PATH = $(objtree)/usr
+ quiet_cmd_headers_install = INSTALL $(INSTALL_HDR_PATH)/include
+       cmd_headers_install = \
+ 	mkdir -p $(INSTALL_HDR_PATH); \
+-	rsync -mrl --include='*/' --include='*\.h' --exclude='*' \
+-	usr/include $(INSTALL_HDR_PATH)
++	find usr/include -type f -name '*.h' -exec sh -c 'set -e; f="$${1\#usr/}"; dd="$(INSTALL_HDR_PATH)/$${f%/*}"; [ -d "$$dd" ] || mkdir -p "$$dd"; cp "usr/$$f" $(INSTALL_HDR_PATH)/"$$f"' -- '{}' ';'
+ 
+ PHONY += headers_install
+ headers_install: headers
```

Comment:
I don't think we need this patch, just follow what LFS do:
```diff
diff --git a/srcpkgs/kernel-libc-headers/template b/srcpkgs/kernel-libc-headers/template
index 715e0be817..09e3456812 100644
--- a/srcpkgs/kernel-libc-headers/template
+++ b/srcpkgs/kernel-libc-headers/template
@@ -26,12 +26,13 @@ esac
 
 do_build() {
 	make mrproper
-	make ARCH=${_arch} headers_check
+	make ARCH=${_arch} headers
 }
 do_install() {
-	make ARCH=${_arch} INSTALL_HDR_PATH=${DESTDIR}/usr headers_install
-	# Remove unused files.
-	rm -f $(find ${DESTDIR}/usr/include -name .install -or -name ..install.cmd)
+	vmkdir usr
+	cp -r usr/include $DESTDIR/usr
+	find $DESTDIR/usr/include -name '.*' -delete
+	rm $DESTDIR/usr/include/Makefile
 	# Remove drm headers, use libdrm.
 	rm -rf ${DESTDIR}/usr/include/drm
 }
```

Notes that `headers_check` is nop, now.

http://www.linuxfromscratch.org/lfs/view/stable/chapter05/linux-headers.html

Reply:
While this template patch could break more easily IMO this is muchhhh much cleaner so i think we should go @sgn 's route.

Reply:
That works for us since we have a package destdir, so I'm okay with it. However, ideally I'd like to have this patch into upstream, if I can get it there.

Reply:
Good luck on sending this patch :-p Meanwhile, I'm inclined to not patch the build system with that long command.

Reply:
FWIW if you want to copy a tree w/o rsync you can achieve similar results with a `{bsd,}tar` pipe like:
```
find usr/include -iname "*.h" | tar cf - -T - | tar xpf - -C $(INSTALL_HDR_PATH)
```
This is untested and may need `find .. -print0` and `tar --null` in case of blanks in path or filenames.